packages/@lwc/engine-server/src/__tests__/fixtures/styles/expected.html

```diff
@@ -1,16 +1,5 @@
 <x-styles>
   <template shadowroot="open">
-    <style type="text/css">
-      :host {
-        color: red;
-      }
-      .foo {
-        color: blue;
-      }
-      .foo .bar {
-        color: green;
-      }
-    </style>
```

Comment:
Hmmm, maybe this is why we were using VNodes for the `<style>` element in native shadow? Still, seems like we don't have a complete solution yet for styling in SSR, so maybe this is okay for now?

Reply:
Based on our discussion of this, falling back to a simpler VNode `<style>` strategy for now to support SSR.